lib/foreman_maintain/concerns/disk_utility.rb

```diff
@@ -0,0 +1,41 @@
+require 'foreman_maintain/concerns/disk_utility/device'
+require 'foreman_maintain/concerns/disk_utility/disk_type'
+require 'foreman_maintain/concerns/disk_utility/disk_io'
+require 'foreman_maintain/concerns/disk_utility/file_io'
+
+module ForemanMaintain
+  module Concerns
+    module DiskUtility
+      include Logger
+      MIN_SPEED_MBPS = 80
+      DEFAULT_DIRS = ['/var/lib/pulp', '/var/lib/mongodb', '/var/lib/pgsql'].freeze
+
+      def self.included(klass)
+        klass.extend(self)
+      end
+
+      def pass?
```

Comment:
> Perhaps, instead of having this as a concern, the logic that this module defines could go directly to the check.

The reason to extract it in a concern(or util) was to be able to unit test it.

If we include everything in check `Checks::DiskSpeedMinimal` then it needs to be functional tested(preferably). Unit testing `Checks::DiskSpeedMinimal` would be feasible? Like initializing checks and testing the instance methods etc.

If it needs to be abstracted out, what would be the best way to do?